login_server/loginserver.py

```diff
@@ -330,6 +330,22 @@ def handle_http_request_message(self, msg):
                 'online_players': len(self.players),
                 'online_servers': len(self.game_servers)
             }, sort_keys=True, indent=4))
+        elif msg.env['PATH_INFO'] == '/detailed_status':
```

Comment:
Is there currently any logging of http requests? We don't protect against DoS right now, but I think we do at least need logging to tell when someone is DoSing.

Reply:
Oh! Good thinking. I'll change this to include logging.